core/test-extension/deployment/src/test/java/logging/ConsoleHandlerTest.java

```diff
@@ -0,0 +1,40 @@
+package logging;
+
+import java.util.Arrays;
+import java.util.logging.Formatter;
+import java.util.logging.Handler;
+import java.util.logging.Level;
+import java.util.logging.LogManager;
+
+import org.jboss.logmanager.formatters.PatternFormatter;
+import org.jboss.logmanager.handlers.ConsoleHandler;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import io.quarkus.runtime.logging.InitialConfigurator;
+import io.quarkus.test.QuarkusUnitTest;
+
+public class ConsoleHandlerTest {
+    @RegisterExtension
+    static final QuarkusUnitTest config = new QuarkusUnitTest()
+            .setArchiveProducer(() -> ShrinkWrap.create(JavaArchive.class)
+                    .addAsResource("application-console-output.properties", "application.properties"));
+
+    @Test
+    public void consoleOutputTest() {
+        LogManager logManager = LogManager.getLogManager();
+        Assertions.assertTrue(logManager instanceof org.jboss.logmanager.LogManager);
+        Handler[] handlers = InitialConfigurator.DELAYED_HANDLER.getHandlers();
```

Comment:
I would also add a check that the `InitialConfigurator.DELAYED_HANDLER` has been added to the root logger `Logger.getLogger("").getHandlers()`.